--- kdebase-3.5.10/admin/acinclude.m4.in~	2010-01-19 00:23:09.000000000 +0200
+++ kdebase-3.5.10/admin/acinclude.m4.in	2010-01-19 00:25:25.115858232 +0200
@@ -4065,7 +4065,7 @@
 
   want_pam=
   AC_ARG_WITH(pam,
-    AC_HELP_STRING([--with-pam[=ARG]],[enable support for PAM: ARG=[yes|no|service name]]),
+    AC_HELP_STRING([--with-pam@<:@=ARG@:>@],[enable support for PAM: ARG=@<:@yes|no|service name@:>@]),
     [ if test "x$withval" = "xyes"; then
         want_pam=yes
         pam_service=kde
