--- kdeadmin-3.0.2/lilo-config/kde/lilo.desktop.orig	2002-07-10 20:47:05.000000000 +0200
+++ kdeadmin-3.0.2/lilo-config/kde/lilo.desktop	2002-07-10 20:47:16.000000000 +0200
@@ -133,7 +133,7 @@
 Keywords[lt]=lilo,boot,bootup,boot manager,linux,Other OS,MBR,startup,start,užkrova,krovimasis,kitos OS
 Keywords[lv]=lilo,sāknēt,sāknēšana,sāknēšanas menedžeris,linux,Citas OS,MBR,startēšana,startēt
 Keywords[nb]=lilo,boot,oppstart,linux,andre operativsystem,MBR,start
-Keywords[nl]=lilo,boot,bootup,boot manager,beheerder,linux,mbr,startup,start, ander besturingssysteem,opstarten,Windows,OS\2,BeOS
+Keywords[nl]=lilo,boot,bootup,boot manager,beheerder,linux,mbr,startup,start, ander besturingssysteem,opstarten,Windows,OS2,BeOS
 Keywords[nn]=lilo,boot,oppstart,linux,andre operativsystem,MBR,start
 Keywords[pl]=lilo,boot,menedżer ładowania systemu,linux,linuks,inne systemy,MBR,start
 Keywords[pt]=lilo,arranque,boot,gestor de arranque,linux,outro SO,MBR,iniciar,início
