diff -Nur kdeadmin-2.0.org/configure.in kdeadmin-2.0/configure.in
--- kdeadmin-2.0.org/configure.in	Mon Oct 16 13:50:36 2000
+++ kdeadmin-2.0/configure.in	Sat Nov  4 13:37:05 2000
@@ -36,7 +36,6 @@
 AC_ARG_PROGRAM
 
 #MIN_CONFIG
-DO_NOT_COMPILE="$DO_NOT_COMPILE kdat"
 AC_PATH_PAM
 # PAMLIBS is used in kuser
 CXXFLAGS="$CXXFLAGS -DQT_CLEAN_NAMESPACE -DQT_NO_COMPAT -DQT_NO_ASCII_CAST"
diff -Nur kdeadmin-2.0.org/configure.in.in kdeadmin-2.0/configure.in.in
--- kdeadmin-2.0.org/configure.in.in	Mon Oct 16 12:22:25 2000
+++ kdeadmin-2.0/configure.in.in	Sat Nov  4 13:25:17 2000
@@ -1,5 +1,4 @@
 #MIN_CONFIG
-DO_NOT_COMPILE="$DO_NOT_COMPILE kdat"
 AC_PATH_PAM
 # PAMLIBS is used in kuser
 CXXFLAGS="$CXXFLAGS -DQT_CLEAN_NAMESPACE -DQT_NO_COMPAT -DQT_NO_ASCII_CAST"
