--- kdeadmin-3.0.3/kdat/KDat.cpp--	2002-09-03 15:42:48.000000000 +0200
+++ kdeadmin-3.0.3/kdat/KDat.cpp	2002-09-03 15:44:11.000000000 +0200
@@ -234,6 +234,15 @@ KDat::~KDat()
     if ( Options::instance()->getLockOnMount() ) {
         TapeDrive::instance()->unlock();
     }
+    delete _tapeDriveMenu;
+    delete _archiveMenu;
+    delete _mountedArchiveMenu;
+    delete _mountedTapeFileMenu;
+    delete _localFileMenu;
+    delete _tapeMenu;
+    delete _backupProfileRootMenu;
+    delete _backupProfileMenu;
+
 }
 
 void KDat::popupTapeDriveMenu( const QPoint& p )
